Bodekaer/Labster.EdX
# WEB-2367: Remove the help button that shows when a user is logged in.

**Description:** CS team do not use Uservoice anymore -> remove help button
 
**JIRA:** https://liv-it.atlassian.net/browse/WEB-2367

**Testing instructions:**

https://edx-eucalyptus.labster.com

1. Open the sandbox
2. Login
3. Make sure Help buton is not shownd.

**Reviewers:**

**Merge checklist:**

**Post merge:**
